.gitignore

```diff
@@ -0,0 +1 @@
+*.class
```

Comment:
You should put some exception below in .gitignore
```suggestion
*.o
*.out
*.exe 
```

Reply:
Thanks for your feedback. I'll work on it